symplyphysics/symbols/basic.py

```diff
@@ -7,9 +7,14 @@
 
 from sympy.physics import units
 from sympy.physics.units.definitions.dimension_definitions import angle as angle_type
-from symplyphysics.core.dimensions import dimensionless
+from symplyphysics.core.dimensions import dimensionless, any_dimension as _any_dimension
 from symplyphysics.core.symbols.symbols import SymbolNew
 
+any_dimension = SymbolNew("X", _any_dimension)
```

Comment:
Wrong name. I suggest `any_quantity` or `anything`.

Reply:
You're right, didn't think of that then.